nautobot/extras/forms/forms.py

```diff
@@ -1246,11 +1246,18 @@ class JobQueueBulkEditForm(TagsBulkEditFormMixin, NautobotBulkEditForm):
         queryset=Tenant.objects.all(),
         required=False,
     )
+    secrets_group = DynamicModelChoiceField(
+        queryset=SecretsGroup.objects.all(),
+        required=False,
+    )
+    context = forms.CharField(required=False, max_length=CHARFIELD_MAX_LENGTH)
     description = forms.CharField(required=False, max_length=CHARFIELD_MAX_LENGTH)
 
     class Meta:
         model = JobQueue
         nullable_fields = [
+            "context",
```

Comment:
It doesn't look like `context` is `null=True` (nor should it be) so is this wrong?